### ISSUE TITLE
Provided type not conforming to regular expression 

### ISSUE THREAD

**Issue by bergstar** [phpro/soap-client#158]
Config generated with same input as shown in demo. By running:
`./vendor/bin/soap-client generate:client --config=config/soap-client.php`

I get this error:

```
Provided type "App\Type\" is invalid: must conform "/^[a-zA-Z_\x80-\xff][a-zA-Z0-9_\x80-\xff]*(\\[a-zA-Z_\x80-\xff][a-zA-Z0-9_\x80-\xff]*)*$/"
```

PHP 7.2.5

**Comment by veewee** [comment]
Hi @bergstar,

This error is triggered by zend-code and means that a XSD complexType is not a valid classname in PHP. Normally this should be normalized by this package.

Can you provide us a WSDL and your config file to check what went wrong?

**Comment by bergstar** [comment]
Hi @veewee ,

Thank you for reply. I have shared WSDL and config file with you by invitation.

**Comment by veewee** [comment]
Thanks for reporting @bergstar!
I've created a fix in #160.

**Comment by bergstar** [comment]
@veewee Thank you so much for the solution!
